ios/FluentUI/Table View/TableViewCellFileAccessoryView.swift

```diff
@@ -84,6 +84,17 @@ open class TableViewCellFileAccessoryView: UIView {
         }
     }
 
+    /// The minimum count of actions.
```

Comment:
I think it's important to mention on the documentation here that this property is used to force the width of the accessory view by adding shimming empty views to it. It will also help explain why the high value may cause a broken layout.

And maybe a recommendation like: "use this property when you have variable action counts across cells' accessory views and need to have them all aligned".